Feat: add title query params for exported csv and xlsx + download translations (#872)
gristlabs/grist-core: README.md

GRIST_HTTPS_PROXY   | if set, use this proxy for webhook payload delivery.
GRIST_ID_PREFIX | for subdomains of form o-*, expect or produce o-${GRIST_ID_PREFIX}*.
GRIST_IGNORE_SESSION | if set, Grist will not use a session for authentication.

> GRIST_INCLUDE_CONTEXT_TO_DOWNLOAD_FILENAMES | if set to true the organization name and the worspace name will be added to downloaded filenames, by default is set to false.

Reviewer: ```suggestion
GRIST_INCLUDE_CONTEXT_TO_DOWNLOAD_FILENAMES | if set to true, the organization and the workspace names are added to the downloaded Grist file names, defaults to `false`.
```

(Not 100% sure)
